sqlx-core/src/query_builder.rs

```diff
@@ -139,6 +139,28 @@ where
     /// [`.push_bind()`][Separated::push_bind] methods which push `separator` to the query
     /// before their normal behavior. [`.push_unseparated()`][Separated::push_unseparated] is also
     /// provided to push a SQL fragment without the separator.
+    ///
+    /// ```rust
+    /// # #[cfg(feature = "mysql")] {
+    /// use sqlx::{Execute, MySql, QueryBuilder};
+    /// let foods = vec!["pizza".to_string(), "chips".to_string()];
+    /// let mut query_builder: QueryBuilder<MySql> = QueryBuilder::new(
+    ///     "SELECT * from food where name in ("
+    /// );
+    /// // One element vector is handled correctly but an empty vector
+    /// // would cause a sql syntax error
+    /// let mut separated = query_builder.separated(", ");
+    /// for value_type in filtered_types.iter() {
```

Comment:
`filtered_types` is not defined: https://github.com/launchbadge/sqlx/runs/7350735897?check_suite_focus=true#step:5:701